docs/content-crag/concepts/solids-pipelines/jobs.mdx

```diff
@@ -0,0 +1,156 @@
+---
+title: Jobs | Dagster
+description: A job is an executable particularization of a graph that binds it to resources and configuration.
+---
+
+# Jobs
+
+A job is an executable particularization of a graph that binds it to resources and configuration.
+
+## Relevant APIs
+
+| Name                                                  | Description                                                                                                                                                                                                    |
+| ----------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| <PyObject object="GraphDefinition" method="to_job" /> | The method used to create a job from a graph.                                                                                                                                                                  |
+| <PyObject object="JobDefinition" />                   | A job definition.  Includes a graph, [resources](/concepts/resources), default [configuration](/concepts/configuration/config-schema), tags, [hooks](/concepts/solids-pipelines/solid-hooks), and an executor. |
+
+Dagster is built around the observation that any data DAG typically contains a logical core of transformation (the graph), which is reusable across a set of environments or scenarios. The graph needs to be "particularized" for each environment, usually by plugging in configuration and services that are specific to that environment.
+
+A Dagster _job_ is the combination of a graph and a set of [resources](/concepts/resources) and [configuration](/concepts/configuration/config-schema).  You can create multiple jobs from the same graph.  For example, you might build a development job that executes a graph against sample data and stores outputs in local files, as well as production job that executes the same graph against production data and stores outputs in a cloud data lake.
+
+Jobs are the unit of execution and monitoring in Dagster deployments - the [Dagit UI](/concepts/dagit/dagit) centers on jobs.  Dagit makes it easy to view all the runs of each job in one place, as well as to manually kick off runs for a job.
+
+[Schedules](/concepts/partitions-schedules-sensors/schedules) and [sensors](/concepts/partitions-schedules-sensors/sensors) target jobs.  You can also launch jobs manually from Dagit, GraphQL APIs, or the command line.
+
+## Automatic coercion of graphs to jobs
+
+The simplest way to create a job is to do nothing at all.  I.e. you can include a graph in a repository, or target it from a schedule or sensor, and Dagster will automatically coerce it into a job.
```

Comment:
While I like the definitiveness of this sentence, I think that it is potentially confusing, because not every graph can actually be coerced into a job (those that need resources). Maybe 
`If a graph does not have any external dependencies (resources), then it can be thought of as a job as well. You can include these graphs in a repository, target them from schedules and sensors, and Dagster will automatically treat them as jobs.`